Jenkinsfile

```diff
@@ -72,7 +72,7 @@ dist = new BuildConfig()
 dist.nodetype = 'linux'
 dist.name = 'dist'
 dist.build_cmds = [
-    "${CONDA_INST} numpy",
+    "${CONDA_INST} numpy=1.15.2",
```

Comment:
Is it necessary to specify a version here given that it is specified in the requirements above?
Also, isn't numpy pinned to 1.14 at the beginning of the file?
I don't understand enough of this, perhaps @mrendina can help.

Reply:
This is for the building of just the `jwst` package, which is the only thing done here.  It's actually probably not necessary to specify numpy version here, as I doubt `1.15.3`  breaks the build process.